src/core/views.py

```diff
@@ -45,15 +51,29 @@ class GalleryView(ListSortMixin, generic.ListView):
     paginate_by = 12
 
     def get_queryset(self):
-        return self.get_sorted_queryset() \
-            .filter(published=1) \
-            .select_related('dataset')
+        return (self.get_sorted_queryset()
+                .filter(published=1)
+                .select_related('dataset'))
 
 
 class HelpView(generic.TemplateView):
     template_name = "site/help.html"
 
 
+class RecoverInvalid(Recover):
+    """Override password_reset.Recover so that invalid
```

Comment:
@ldko, that is actually suppose to be a python path. **But** after you pointed that out, I noticed that the python path is incorrect. It should `password_reset.views.Recover`.